### DISCUSSION TITLE
--datalayer-port unkown option

### DISCUSSION THREAD

**Discussion by hubsmith** [tinacms/tinacms#4193] [Help]
Hi, wanted to change the default datalayer port of tinacms since it's already used by php-fpm. However when I use the option with this command:

`tinacms dev --datalayer-port 9900 -c "yarn start" `

I get an unknown option error, the argument is in this documentation: https://tina.io/docs/cli-overview/

Did I miss anything or is my syntax wrong? Thank you to those who are willing to help.

**Comment by kldavis4** [comment]
This should work: ` tinacms dev --datalayer-port 9900 -c [COMMAND]`

What version of @tinacms/cli are you using?

**Comment by hubsmith** [comment]
Got it working, thank you for the clue! I had to update tinacms to the latest version and the --datalayer-port option is now valid.
